crates/stackable-operator/src/kvp/label/selector.rs

```diff
@@ -1,23 +1,37 @@
 use k8s_openapi::apimachinery::pkg::apis::meta::v1::LabelSelector;
+use snafu::Snafu;
+
+type Result<T, E = SelectorError> = std::result::Result<T, E>;
+
+#[derive(Debug, PartialEq, Snafu)]
+pub enum SelectorError {
+    #[snafu(display("labelSelector with binary operator {operator:?} must have values"))]
```

Comment:
nitpick: `labelSelector` should be two separate words.

```suggestion
    #[snafu(display("label selector with binary operator {operator:?} must have values"))]
```

Also applies to variants below.